client/internal/vanus/store/block_store.go

```diff
@@ -91,21 +207,98 @@ func (s *BlockStore) Append(ctx context.Context, block uint64, event *ce.Event)
 	if err != nil {
 		return -1, err
 	}
-	// TODO(Y. F. Zhang): batch events
 	return res.GetOffsets()[0], nil
 }
 
+func (s *BlockStore) AppendManyStream(ctx context.Context, block uint64, events []*ce.Event) ([]int64, error) {
+	_ctx, span := s.tracer.Start(ctx, "AppendManyStream")
+	defer span.End()
+
+	var (
+		err  error
+		resp *segpb.AppendToBlockStreamResponse
+	)
+
+	if s.appendStream == nil {
```

Comment:
always get appendStream by `s.connectAppendStream`.

Reply:
this is just a blank check, i think use s.connectAppendStream() is not a good idea